src/components/ModalDialog/ModalDialog.js

```diff
@@ -1,57 +1,39 @@
 import React from "react";
-import "font-awesome/css/font-awesome.min.css";
-import "./ModalDialog.css";
+import Icon from "../Icon";
 import PropTypes from "prop-types";
 import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from "reactstrap";
 
-function ModalDialog(props) {
-  let header = props.header || "Alert";
-  let iconStyle = props.iconStyle || "Warning";
-  let cancelPrompt = props.cancelPrompt || "Cancel";
-  let confirmPrompt = props.confirmPrompt || "Confirm";
-
+function ModalDialog({
```

Comment:
Using destructuring to shorten calls below.

Reply:
Note that I eliminated the need for the default boilerplate below by declaring defaultProps.

Reply:
Added default props